### ISSUE TITLE
The `validate-checksum` task fails with `task` v3.40.0 and higher.

### ISSUE THREAD

**Issue by kirkrodrigues** [y-scope/yscope-dev-utils#14]
The `validate-checksum` [task](https://github.com/y-scope/yscope-dev-utils/blob/fe2fb1b814636db2c171a43da0b95d92342162b0/taskfiles/utils.yml#L39) fails with `task` v3.40.0 since v3.40.0 changes the behaviour of how undefined variables are treated in the [`ref`](https://github.com/y-scope/yscope-dev-utils/blob/fe2fb1b814636db2c171a43da0b95d92342162b0/taskfiles/utils.yml#L53) attribute.

One potential fix is to use an empty list as the `ref`'s default:

```yaml
          EXCLUDE_PATHS:
            ref: "default (list) .EXCLUDE_PATHS"
```

**Comment by kirkrodrigues** [comment]
Fixed by [ad576e4](https://github.com/y-scope/yscope-dev-utils/commit/ad576e43c1a43d7a6afde79fc9c3c952b7bf28bd).

**Comment by kirkrodrigues** [comment]
Reported bug to task: https://github.com/go-task/task/issues/1911
